nixos/modules/virtualisation/vmware-guest.nix

```diff
@@ -25,6 +25,8 @@ in
         serviceConfig.ExecStart = "${open-vm-tools}/bin/vmtoolsd";
       };
 
+    environment.etc."vmware-tools".source = "${pkgs.open-vm-tools}/etc/vmware-tools/*";
```

Comment:
We mostly don't like such things. Why do you need that and can't it be avoided with patching some sources?